python/ray/tests/test_cli.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import re
+
+import pytest
+
+import ray.scripts.scripts as scripts
+from click.testing import CliRunner
+
+def test_ray_start():
+    runner = CliRunner()
+    result = runner.invoke(
+        scripts.start,
+        ["--head", "--log-new-style"])
+    assert runner.invoke(scripts.stop).exit_code == 0
+
+    expected_lines = [
+        r"Local node IP: .+",
+        r"Available RAM",
+        r"  Workers: .+ GiB",
+        r"  Objects: .+ GiB",
+        r"",
+        r"  To adjust these values, use",
+        r"    ray\.init\(memory=<bytes>, object_store_memory=<bytes>\)",
+        r"Dashboard URL: .+",
+        r"",
+        r"--------------------",
```

Comment:
question - is it appropriate to do this or is it better to have this in a text file?